ntdll: load the FEX ARM64EC bootstrap before the first emulated call.

diff --git a/dlls/ntdll/loader.c b/dlls/ntdll/loader.c
--- a/dlls/ntdll/loader.c
+++ b/dlls/ntdll/loader.c
@@ -4412,6 +4412,8 @@ static void load_arm64ec_module(void)
         ERR( "could not load %s, status %lx\n", debugstr_w(module), status );
         NtTerminateProcess( GetCurrentProcess(), status );
     }
+
+    update_load_config( wm->ldr.DllBase );
 }
 
 #endif
diff --git a/dlls/ntdll/signal_arm64ec.c b/dlls/ntdll/signal_arm64ec.c
--- a/dlls/ntdll/signal_arm64ec.c
+++ b/dlls/ntdll/signal_arm64ec.c
@@ -226,6 +226,10 @@ NTSTATUS arm64ec_process_init( HMODULE module )
     info->EmulatedMachineType = IMAGE_FILE_MACHINE_AMD64;
     memcpy( KiUserExceptionDispatcher_orig, KiUserExceptionDispatcher_thunk, sizeof(KiUserExceptionDispatcher_orig) );
 
+    __os_arm64x_check_call = arm64x_check_call;
+    __os_arm64x_check_icall = arm64x_check_call;
+    __os_arm64x_check_icall_cfg = arm64x_check_call;
+
     enter_syscall_callback();
     if (pProcessInit) status = pProcessInit();
     if (!status)
@@ -236,9 +240,6 @@ NTSTATUS arm64ec_process_init( HMODULE module )
     }
     if (!status && pThreadInit) status = pThreadInit();
     leave_syscall_callback();
-    __os_arm64x_check_call = arm64x_check_call;
-    __os_arm64x_check_icall = arm64x_check_call;
-    __os_arm64x_check_icall_cfg = arm64x_check_call;
     return status;
 }
 
